src/main.cpp

```diff
@@ -1075,7 +1075,7 @@ QString UsageArguments ( char** argv )
            "  -L, --licence         show an agreement window before users can connect\n"
            "  -m, --htmlstatus      enable HTML status file, set file name\n"
            "  -o, --serverinfo      registration info for this Server.  Format:\n"
-           "                        [name];[city];[country as Qt5 QLocale ID]\n"
+           "                        [name];[city];[country as two-letter code or Qt5 QLocale ID]\n"
```

Comment:
Should we say "two letter ISO code"?  Otherwise people still might not know whether to use "UK" or "gb" or whatever.

Reply:
Done.